src/scripts/destinyTrackerApi/trackerErrorHandler.js

```diff
@@ -0,0 +1,25 @@
+class trackerErrorHandler {
```

Comment:
This is another one that's just functions, not an object. No need to instantiate or inject it, just import the functions.